org-formation/700-aws-sso/_tasks.yaml

```diff
@@ -58,10 +58,26 @@ Parameters:
     Type: String
     Default: '906769aa66-023cdfaa-15f0-43d9-b823-b74358ceffd7'
 
+  workflowNextflowDevDeveloperGroup:
+    Type: String
+    Default: 'TBD'
+
+  workflowNextflowDevViewerGroup:
+    Type: String
+    Default: 'TBD'
+
   workflowNextflowProdAdminGroup:   #JC aws-workflow-nextflow-prod-admin
     Type: String
     Default: '906769aa66-5c67ab0f-1091-48f9-b5b9-37d8b99a446a'
 
+  workflowNextflowProdDeveloperGroup:
```

Comment:
where is the corresponding `SsoWorkflowNextflowProdDeveloper` resource?

Reply:
I don't think that's appropriate in prod, so I'm going to remove this bit.